### PR TITLE
Prepare beta release

### PR DESCRIPTION
**Pull request by tjann**
Add changelog, ~minor README changes

### CODE REVIEW

**Comment by tjann** [comment]
I intend to cherry pick the readme change into the beta branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## 1.0.0-beta
+
+**Date** - 8/19/2019
+
+**Release Tag** - [v1.0.0-beta](https://github.com/datacommonsorg/api-r/releases/tag/v1.0.0-beta)
+
+**Release Status** - Current head of branch [`1.0.0-beta`](https://github.com/datacommonsorg/api-r/tree/1.0.0-beta)
+
+Initial and beta release of the R API Client.
```
Comment (beets) [inline CHANGELOG.md]: Could you add a description here, mirroring the python version? (list of added functions, notebook, etc)

Comment (tjann) [reply CHANGELOG.md]: Ah, ok. I mirrored the initial release note from Python. Is bbd94c3 better?

Comment (beets) [reply CHANGELOG.md]: yes, that looks better but it's not in this PR?

Comment (tjann) [reply CHANGELOG.md]: resolved offline, ^ was a stale comment

**Pull request merged**